code_test.rb

```diff
@@ -21,4 +21,16 @@ def test_jump_null
   def test_jump_JNE
     assert_equal("101", @code.jump("JNE"))
   end
+
+  def test_comp_NOT_M
+    assert_equal("110001", @code.comp("!M"))
+  end
+
+  def test_comp_D_MINUS_1
+    assert_equal("001110", @code.comp("D-1"))
+  end
+
+  def test_comp_0
+    assert_equal("101010", @code.comp("0"))
+  end
```

Comment:
Oops, I missed that these tests aren’t correct — `#comp` is supposed to return seven bits, not six. Fortunately you’ve fixed this in #2.